### ISSUE TITLE
[BUG-Critical] execution view does not show linked issues

### ISSUE THREAD

**Issue by steirer** [kawasima/redmine_impasse#187]
- user has found a bug in version v1.(custom redmine field "found in version")
- developer fixed the bug in version v2 (target version)
- developer creates a testcase for this issue
- tester creates a testplan for version v2 (target version)
- tester executes the test case

result
- system does NOT show the related bug
  => this is a big problem. Because a lot of testing is about testing if a certain bug is fixed ... and setting it to "Closed". This is not possible.

**Comment by bjritchey** [comment]
I use this feature by creating a test case when I discover a bug, then I fail the test case which generates a linked bug - bug screen opens and I create the bug.  This usually results in a link to the test case on the bug and a link to the bug on the test case.  No longer working.

**Comment by cforce** [comment]
> I fail the test case which generates a linked bug - bug screen opens and I create the bug

For the the bug form does not open any more, if i tag a test as failed. Is that featured dropped?
Else i have the same problem.

**Comment by steirer** [comment]
Right. and there's a difference between linking "requirements" to a testcase and linking issues to testcases via this bug-creation-functionality.  Actually this might be a good idea anyway:
- requirements linking: these links are permanent. like a certain feature is tested by serveral test cases. cool.
- bug linking: this only works via the bug creation mechanism when you set a test case result for "NG".
  But here is the big problem: you cannot link an existing issue and you don't see the link in the test execution view! and this is really a problem.
  My workaround is to put the issue number into the "notes" field ...at least I don't loose the connection.

**Comment by cforce** [comment]
Fully agree. Linking existing issues shall be possible also.

For implementation of the "NG" auto isse creation i urgent recommend to not reimplement the issue from view directly there in place , because you can easily get into "required user defined issue fields" problems if you forget that extra fields.
Furthermore the layouting in place is a big issue and doesn't make to much sense as place is to limited for a full bug form with unknown number of extra fields. It is absolutly ok to show a regular link in NG case, the user can click to open a new browser tab with a prefilled standard bug request form  using redmine standard new issue view. Don't try to create fat client like ui or double view implemetations. Accept and honor that ur using a web technology. Web is all about linking and urls, so use the standard redmine issue form but pass bug data in html when calling the rest url, so bug its simply prefilled with the test case data needed for bug report, like tracker (is already configurable), topic, description, category (shall be confiurable), link to test case and vice versa must be created.

A coberage report about which requriements has a test case or not, and which test cases has still open bugs is very useful also.

**Comment by cforce** [comment]
fixed - see 
https://github.com/kawasima/redmine_impasse/pull/209

dupe of #178 and #186
